src/main/scala/nl.knaw.dans.easy.sword2/servlets/EasySword2Servlet.scala

```diff
@@ -19,12 +19,12 @@ import java.io._
 import javax.servlet._
 import javax.servlet.http._
 
-class HelloServlet extends HttpServlet {
+class EasySword2Servlet extends HttpServlet {
   @throws(classOf[ServletException])
   @throws(classOf[IOException])
-  override def doGet(request: HttpServletRequest, response: HttpServletResponse) {
+  override def doGet(request: HttpServletRequest, response: HttpServletResponse): Unit = {
     val out: PrintWriter = response.getWriter
-    out.println("hello, world")
+    out.println("EASY Sword2 Service running...")
```

Comment:
great change.
As an improvement, would it be possible to add the version number as well?
See also https://github.com/DANS-KNAW/easy-deposit-api/blob/f3467c347dd103b5fda4b5e35822bb09e795b360/src/main/scala/nl.knaw.dans.easy.deposit/servlets/EasyDepositApiServlet.scala#L28.

If you feel like it, add it to the other services as well 😜

Reply:
Sure, i will look into it

Reply:
Created an issue and corresponding pull requests